tests/test_plugins_unix_config_filesystem.py

```diff
@@ -65,7 +70,7 @@ def __exit__(self, _, __, ___):
 
 
 def test_parse_functions(target_unix: Target, etc_directory: VirtualFilesystem):
-    config_fs = ConfigurationFs(target_unix)
+    config_fs = ConfigurationFilesystem(target_unix)
     entry: ConfigurationEntry = config_fs.get("/new/path/config", collapse=True)
 
     assert entry.parser_items["help"] == "you"
```

Comment:
Could we make a nicer way to get configuration values? Perhaps just make the `entry` itself behave like a dictionary with `__item__` and `.get()`?

Reply:
You _could_, but instead of raising a `NotADirectoryError` you'd just be returning the actual configuration value.

Reply:
Sorry, I probably meant that instead of letting end values be entries as well, that those just return the actual values, instead of having to use `.open()`.

Maybe I'm still just unsure if the filesystem approach is a good UX/API experience.